smoke-test/tests/tokens/revokable_access_token_test.py

```diff
@@ -56,8 +133,8 @@ def test_admin_can_create_list_and_revoke_tokens():
     assert res_data["data"]
     assert res_data["data"]["listAccessTokens"]["total"] is not None
     assert len(res_data["data"]["listAccessTokens"]["tokens"]) == 1
-    assert res_data["data"]["listAccessTokens"]["tokens"][1]["actorUrn"] == "urn:li:corpuser:datahub"
-    assert res_data["data"]["listAccessTokens"]["tokens"][1]["ownerUrn"] == "urn:li:corpuser:datahub"
+    assert res_data["data"]["listAccessTokens"]["tokens"][0]["actorUrn"] == "urn:li:corpuser:datahub"
```

Comment:
Interesting- why was this previously set at 1?

Reply:
It was a typo from a previous version of this test.